### PR TITLE
update to keras 3 - Sequential API

### PR DESCRIPTION
**Pull request by takumiohym**
Update Summary
- Updated code and markdown descriptions to Keras 3
- Added descriptions on model.fit
- modified tf.data and training parameters a bit for better consistency with Functional API notebook https://github.com/GoogleCloudPlatform/asl-ml-immersion/pull/624
- Added detailed descriptions on Vertex AI serving

### CODE REVIEW

File: notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex.ipynb [hunk 1]
```diff
@@ -50,10 +50,8 @@
    "outputs": [],
```
Comment (sanjanalreddy) [inline notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex.ipynb:50]: <p>the documentation link for losses needs to be updated, it's pointing to the tf losses right now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/629/discussion/#comment-2229926440'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='16'/>

Comment (sanjanalreddy) [inline notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex.ipynb:50]: <p>This cell throws a NameError because OUTDIR needs to be initialized. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/629/discussion/#comment-2229926451'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='22'/>

Comment (sanjanalreddy) [inline notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex.ipynb:50]: <p>The RMSE doesn't get better here which is expected. For better understanding, can we perhaps put a note in markdown that the loss not improving is expected and we will work on improving it in the next notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/629/discussion/#comment-2229926455'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='28'/>

Comment (takumiohym) [reply notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex.ipynb:50]: <p>Makes sense. Added!</p>

Comment (takumiohym) [reply notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex.ipynb:50]: <p>Thanks for catching this! Fixed.</p>

Comment (takumiohym) [reply notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex.ipynb:50]: <p>Fixed! Thank you.</p>
